### DISCUSSION TITLE
Welcome to `{lactater}` Discussions!

### DISCUSSION THREAD

**Discussion by fmmattioni** [fmmattioni/lactater#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Klagne** [comment]
Hi!
First of all, thanks a lot for sharing this tool. I actually tried to build something similar for swimming.

I’m a French swimming coach based in a small town. I’m passionate about the sport — I trained intensely for several years, but without reaching the performance level I had hoped for. That experience led me to study sport sciences, with a focus on the psychological aspects, and eventually complete a PhD in this field.

My goal is to offer the very best to my athletes, which is why I wanted to use lactate testing to improve the effectiveness of my training sessions. So here I am!

If you have any questions, I’d be happy to answer them and share my experience with you.
